bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/BulkGraviton_VBF_WW_inclu/BulkGraviton_VBF_WW_inclu_narrow_M2000/BulkGraviton_VBF_WW_inclu_narrow_M2000_proc_card.dat

```diff
@@ -0,0 +1,13 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set gauge unitary
+set complex_mass_scheme False
+import model RS_bulk_ktilda
+define p = u c d s u~ c~ d~ s~
```

Comment:
remove proton and jet definition as is picked up automatically based on the import model.
by the way, is this really a 4F without gluon?!?

Reply:
Ok, removed these two definitions.